content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -61,6 +61,11 @@ If `startingDeadlineSeconds` is set to a large value or left unset (the default)
 and if `concurrencyPolicy` is set to `Allow`, the jobs will always run
 at least once.
 
+{{< caution >}}
+If `startingDeadlineSeconds` is set to a value less than 10 seconds, the CronJob may not be scheduled. This is because the CronJob controller checks things every 10 seconds. As an alpha feature, there is a V2 controller implementation with different behavior. The version 2 controller provides experimental performance improvements.
```

Comment:
Hi @salva-gglez . Thanks for opening this pull request.
The first part of this caution section looks good. Do the last two sentences add value to the section?
What specific changes in the CronJob v2 alpha version (https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#cronjob-v2alpha1-batch) should be mentioned here?

Reply:
Hello @kbhawkey. I can't see what implementations details are changing on the new Crontab version. Last sentences do not provide too much value but I think it's worth to mention about the new version and the performance improvements.

Reply:
Does enabling the v2 controller affect the scheduling behavior with a low `startingDeadlineSeconds`?

If yes: document the difference. If not: leave those 2 sentences out, or move them outside of the callout.

Reply:
It's also OK to not mention the v2 controller feature gate at all, and then as this merges, log an issue to investigate whether the documentation needs a tweak.

Reply:
I'll remove the v2 controller mention and I'll log an issue to investigate de new behaviour.

Reply:
@sftim I think it's ok to merge.